projects/plugins/jetpack/modules/wordads/class-wordads.php

```diff
@@ -738,6 +743,12 @@ public function get_ad_snippet( $section_id, $height, $width, $location = '', $c
 			}
 
 			$loc_id = esc_js( $loc_id );
+
+			$format = WordAds_Formats::get_format_slug( (int) $width, (int) $height );
+			if ( $format === '' ) {
```

Comment:
Further down you're ensuring format is a string before comparing so was wondering why not do the same here? `if ( '{$format}' !== '' ) {`